server/.env.example

```diff
@@ -1,7 +1,7 @@
 # General site settings
 APP_ENV=local
 APP_DEBUG=true
-APP_KEY=SomeRandomString
+APP_KEY=SomeRandomStringSomeRandomString
```

Comment:
Is this an accident, or is the key just too short?  If the latter, maybe make that more clear here.